testdata/p4_16_samples/emit_struct.p4

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
I don't understand what this test is here for. The ifdef is false, so this is just an empty file for the compiler. Please delete it or replace it with a useful test. You should also delete all the reference outputs it produces.

Reply:
I have to remove this test.  The PR has does not change default behavior for emit or extract.

Reply:
This file is getting removed.  Since emit and extract behavior is not changed, this file is not needed.